packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -77,7 +85,37 @@ function applyTransaprentBackplateStyles(): CSSRules<DesignSystem> {
                 return disabledContrast(
                     designSystem.contrast,
                     designSystem.foregroundColor,
-                    designSystem.brandColor
+                    designSystem.backgroundColor
+                );
+            },
+            fill: (config: DesignSystem): string => {
+                const designSystem: DesignSystem = withDesignSystemDefaults(config);

```

Comment:
@nalogiudice it seems like we're ending up with a fairly significant amount of duplication for `disabledContrast` (3) and `ensureNormalContrast` (4) and use for defining the design system here (7 times). Can we do the work to pass those values into this function so we can define these once instead of redeclaring this multiple times in this function?

Reply:
@chrisdholt ok I did a lot of clean up. There were a bunch of other things imported or created that weren't being used. A lot cleaner now.